### DISCUSSION TITLE
EGL-Sync without Epic Games Launcher

### DISCUSSION THREAD

**Discussion by raycekar** [legendary-gl/legendary#536] [Q&A]
From my understanding, the egl-sync is strictly syncing games with the locally installed epic games launcher, however, do I even need Epic Games Launcher to have legendary to automatically sync save data?

Syncing cloud saves (compatible with EGL)

**Comment by derrod** [comment]
No, cloud saves are separate from syncing installed games. Right now syncing of saves is not automatic however.

**Comment by raycekar** [comment]
Ok, so say I want to have them sync save data within Legendary only, looks I would need to run "legendary sync-saves" after quitting a game every time?
